### PR TITLE
[Website] Updated Explore data flow & user feedback

### PR DESCRIPTION
**Pull request by FaithDaka**
#### Summary of Changes (What does this PR do?)

- Link users to platform on explore data click
- Add new category on publications to support upload of docs
- Altered highlights backend attributes
- Address user feedback

#### Status of maturity (all need to be checked before merging):

- [x] I've tested this locally
- [x] I consider this code done
- [x] This change ready to hit production in its current state
- [x] The title of the PR states what changed and the related issues number (used for the release note).

#### How should this be manually tested?

- Visit explore data and click _Air Quality Dashboard_ to be routed to the air quality map
- View guides category on publications

#### What are the relevant tickets?

- [WEB-371]
- [WEB-374]

#### Screenshots
image

[WEB-371]: https://airqoteam.atlassian.net/browse/WEB-371?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[WEB-374]: https://airqoteam.atlassian.net/browse/WEB-374?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by Baalmart — COMMENTED** [review]
Thanks @FaithDaka , this PR description is amazing!

I just have some few comments as we continue the process.

@123MwanjeMike 
Appears the deploy preview link is having a 502 bad gateway. See attached screenshot.
Screenshot 2023-03-02 at 11 58 34

File: website/frontend/src/pages/ExploreData/index.js [hunk 1]
```diff
@@ -637,7 +637,7 @@ const ExploreData = () => {
                 </div>
               </div>
               <h6>An interactive air quality analytics platform</h6>
-              <Link to="/explore-data/get-started">
+              <Link to={`${NETMANAGER_URL}`}>
```
[the remark this answers is not in the document]
Comment (FaithDaka) [reply website/frontend/src/pages/ExploreData/index.js:640]: Yes @Baalmart. The env variables change within the different containers (staging and production), that's why we didn't hardcode those links.

**Review by Baalmart — COMMENTED** [review]
Hi @FaithDaka , thanks again for this! :) 

1. More feedback using the attached screenshots
2. What "other feedback" was worked on -- as mentioned in the last bullet in the PR description?
Screenshot 2023-03-02 at 12 07 39
Screenshot 2023-03-02 at 12 09 34
Screenshot 2023-03-02 at 12 10 26

**Review by 123MwanjeMike — APPROVED** [review]
Thanks @FaithDaka 
The new changes look good to me
image

**Comment by FaithDaka** [comment]
@Baalmart Thanks for your view. But I wanted users to stay on the same tab so that it seems like they are viewing the map on the same site (the way Github does it for links). Some people don't like the escaping to another tab for links, so it's better to leave that _new tab_ option in the hands of the users.
And for the links, I didn't add any that's why they are not firing any event.

**Comment by Baalmart** [comment]
Thanks @FaithDaka that makes sense. Quite reasonable. 

I am somehow worried about the loading speed of our legacy platform. Won't people attribute the slow platform load (sometimes 5+ minutes) as a website issue?

Perhaps consider doing this direct loading after sorting out our platform load issues?

Happy to hear your additional thoughts.

**Comment by FaithDaka** [comment]
@Baalmart I don't see how a new tab makes people overlook slow load speeds. 
Example: If a deploy preview takes long to load after click, do you attribute that to Github or did the deploy preview just take long?

A UX perspective should help (_a user should always be in control_):https://uxdesign.cc/linking-to-a-new-tab-vs-same-tab-f88b495d2187#:~:text=If%20a%20user%20prefers%20that,doesn't%20work%20in%20reverse.

**Comment by Baalmart** [comment]
Thanks @FaithDaka !

**Review by Baalmart — APPROVED** [review]
All looks good to me now. Thanks. For the rest of the feedback, we shall address them in due course as the discussions continue in the background.

Thanks again @FaithDaka

**Pull request merged**